frontend/javascripts/navbar.js

```diff
@@ -294,6 +294,7 @@ function Navbar({ activeUser, isAuthenticated, history, isInAnnotationView, hasO
 
   const navbarStyle: Object = {
     padding: 0,
+    overflowX: "auto",
```

Comment:
I've got some good news, adding `white-space: nowrap` here will make all of the calculations and the `min-width` in the `tracing_actions_view.js` obsolete :)

![](https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif)

Reply:
![](https://media.giphy.com/media/d2Z9QYzA2aidiWn6/giphy.gif)